tensor_comprehensions/tc_unit.py

```diff
@@ -204,7 +204,8 @@ def tune_and_store(self, tc_name, inputs, mapping_options, cache_file=""):
         try:
             best_options = self.autotuner.tune(cache_file, tc_name, inputs, options, [options])
             return best_options
-        except RuntimeError:
+        except Exception as e:
+            logger.error('Got exception: {}'.format(e))
```

Comment:
is got the best word here? Maybe Raised?

Reply:
yeah, will correct :) raised sounds better